WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml

```diff
@@ -0,0 +1,199 @@
+    // ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
+    // IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
+    // PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
+    //
+    //*********************************************************
+-->
+
+<Page
+    x:Class="WinUIGallery.ControlPages.SpacingPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:WinUIGallery.DesktopWap.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:WinUIGallery"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    x:Name="compactPage"
+    mc:Ignorable="d">
+    <Page.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.ThemeDictionaries>
+                <ResourceDictionary x:Key="Default">
+                    <ImageSource x:Key="DialogImage">ms-appx:///Assets/Design/Dialog.dark.png</ImageSource>
+                    <ImageSource x:Key="CardsImage">ms-appx:///Assets/Design/Cards.dark.png</ImageSource>
+                </ResourceDictionary>
+                <ResourceDictionary x:Key="Light">
+                    <ImageSource x:Key="DialogImage">ms-appx:///Assets/Design/Dialog.light.png</ImageSource>
+                    <ImageSource x:Key="CardsImage">ms-appx:///Assets/Design/Cards.light.png</ImageSource>
+                </ResourceDictionary>
+            </ResourceDictionary.ThemeDictionaries>
+
+
+            <Style x:Key="GridCardStyle" TargetType="Grid">
+                <Style.Setters>
+                    <Setter Property="HorizontalAlignment" Value="Stretch" />
+                    <Setter Property="MinHeight" Value="68" />
+                    <Setter Property="CornerRadius" Value="{StaticResource OverlayCornerRadius}" />
+                </Style.Setters>
+            </Style>
+
+            <Style x:Key="ValueTextStyle" TargetType="TextBlock">
+                <Style.Setters>
+                    <Setter Property="Margin" Value="16,0,0,0" />
+                    <Setter Property="VerticalAlignment" Value="Center" />
+                </Style.Setters>
+            </Style>
+
+            <Style
+                x:Key="DescriptionTextStyle"
+                BasedOn="{StaticResource CaptionTextBlockStyle}"
+                TargetType="TextBlock">
+                <Style.Setters>
+                    <Setter Property="Grid.Column" Value="2" />
+                    <Setter Property="TextWrapping" Value="Wrap" />
+                    <Setter Property="VerticalAlignment" Value="Center" />
+                </Style.Setters>
+            </Style>
+
+            <Style x:Key="VisualBorderStyle" TargetType="Border">
+                <Style.Setters>
+                    <Setter Property="Grid.Column" Value="1" />
+                    <Setter Property="CornerRadius" Value="{StaticResource ControlCornerRadius}" />
```

Comment:
```suggestion
```

Reply:
I think adding a bit of rounding makes it feel more inline with all the other UI elements